README.md

```diff
@@ -327,22 +327,14 @@ The following resources are available for usage in your recipes:
   ```ruby
   require 'ilo-sdk'
 
-  ilo1 = {
-    host: 'ilo1.example.com',  # Required. IP or hostname
-    user: 'Administrator',     # Optional. Defaults to 'Administrator'
-    password: 'secret123',     # Required
-    ssl_enabled: false         # Optional
-  }
-
-  client = ILO_SDK::Client.new(
-    host: ilo1[:host],
-    user: ilo1[:user],
-    password: ilo1[:password],
-    ssl_enabled: ilo1[:ssl_enabled]
+  ilo1 = ILO_SDK::Client.new(
+    host: 'ilo1.example.com',
+    user: 'Administrator',
+    password: 'secret123'
   )
 
```

Comment:
The Chef resource has a property:
``` ruby
property :ilo, Hash
```
In your above changes you have ilo1 as a `ILO_SDK::Client` class. Will this still work when it needs to be a Hash?